doc/build/build_options.md

```diff
@@ -114,6 +115,16 @@ TBB plus more:
   primitive creation time. At the primitive execution time, the threadpool is
   responsible for balancing the static decomposition from the previous item
   across available worker threads.
+  
+#### Vendor BLAS libraries
+oneDNN can use a standard BLAS library for GEMM operations. The `DNNL_BLAS_VENDOR` build option controls BLAS library selection, and defaults to `NONE`. For AArch64 builds with GCC, use the [Arm Performance Libaries](https://developer.arm.com/tools-and-software/server-and-hpc/downloads/arm-performance-libraries):
+
+~~~sh
+$ cmake -DDNNL_BLAS_VENDOR=ARMPL ..
+~~~
+
+Additional options available for development/debug purposes: `MKL` to select [Intel Math Kernel Library](https://software.intel.com/content/www/us/en/develop/tools/math-kernel-library.html); `OPENBLAS` to select the [OpenBLAS library](https://www.openblas.net); and `ANY` to allow CMake to search for a suitable BLAS library. These options are subject to change without notice - please avoid using them.
```

Comment:
In my opinion, since we don't advertise using those, I think it should be enough to just mention there are other BLAS implementation the library can link with. However these configurations are not tested and provided solely for development/debug purposes. Maybe refer to the file [`cmake/options.cmake`](https://github.com/oneapi-src/oneDNN/blob/master/cmake/options.cmake).
No need to list the libraries themselves, as we might extend and shrink the list, and might forget to adjust this place.

Reply:
No problem - updated the commit as suggested.